nemo/collections/nlp/models/language_modeling/megatron_bert_model.py

```diff
@@ -359,21 +375,22 @@ def training_step(self, batch, batch_idx):
         if self.cfg.precision == 16:
             loss_scale = self.trainer.precision_plugin.scaler._scale
             if loss_scale is not None:
-                self.log('loss_scale', loss_scale)
+                self.log('loss_scale', loss_scale, batch_size=1)
```

Comment:
what is this `batch_size=1` do here?

Reply:
It throws an error otherwise. Pretty much pulled it from the gpt micro batch loader code . @erhoo82  might know better.

Reply:
It's a hack to make PTL happy, we need to raise an issue with them.